### PR TITLE
bugfix(navbar): show all tabs when switching to mobile version(low width)

### LINKED ISSUE #3109: Navbar shriking hides navbar options
Navbar shrinking currently hides tabs such as "Create" before switching to mobile version. @cryptodamsky just tried presenting from 13 inch macbook and couldn't click on "Create" because it wasn't there.
image

### PR DESCRIPTION
**Pull request by zhengow**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #3109 
- [ ] Requires deployment <rubick/magick_issue>
- [x] show all tabs when switching to mobile version(with low width)

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [x] I've tested it at </about>
- [x] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=EgCzZLpVe7zg7RHh79qPyC8UrykVkA22YY2oj7sWMZf15Ws)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

https://user-images.githubusercontent.com/38389586/173215112-f8cee62f-e45f-4f5f-a7e9-c9430c924b59.mp4

### CODE REVIEW

**Comment by roiLeo** [comment]
#2760, #2769 related

**Comment by zhengow** [comment]
> #2760, #2769 related

Is my PR ok? Anything need to modify?

File: components/Navbar.vue [hunk 1]
```diff
@@ -337,6 +337,10 @@ export default class NavbarMenu extends mixins(PrefixMixin) {
     }
   }
 
+  .koda-is-hidden-desktop {
+    background-color: $primary !important;
+  }
```
Comment (roiLeo) [inline components/Navbar.vue:342]: this isn't behing used I think you can remove it

**Review by roiLeo — COMMENTED** [review]
I prefer this solution instead of hiding links according to breakpoints

note: I would remove unused/commented css

**Comment by zhengow** [comment]
> I prefer this solution instead of hiding links according to breakpoints
> 
> 
> 
> note: I would remove unused/commented css 

ok

File: components/Navbar.vue (hunk 1 above)
Comment (zhengow) [reply components/Navbar.vue]: done 👍

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 48.21 USD/KSM ~ 1.037 $KSM
🧗 [EgCzZLpVe7zg7RHh79qPyC8UrykVkA22YY2oj7sWMZf15Ws](https://kusama.subscan.io/account/EgCzZLpVe7zg7RHh79qPyC8UrykVkA22YY2oj7sWMZf15Ws)
🔗 [0x9a244178f9c9b845d8a1a72c09732aae25742aacee532d2bfb66dee6f5c35193](https://kusama.subscan.io/extrinsic/0x9a244178f9c9b845d8a1a72c09732aae25742aacee532d2bfb66dee6f5c35193)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**